README.md

```diff
@@ -1,3 +1,4 @@
+![GitHub issues](https://img.shields.io/github/issues/tanvimoharir/NOvid-20) ![GitHub pull requests](https://img.shields.io/github/issues-pr/tanvimoharir/NOvid-20)
```

Comment:
Over here you have added the issues and issues-pr badges for your forked repo. If you could change it to main repo it would be good.

Reply:
I have changed this.